scripts/ci-helper.sh

```diff
@@ -166,48 +75,37 @@ function build() {
     ldflags=""
   fi
 
-  ldflags="${ldflags}-X github.com/hashicorp/vault/version.Version=$version -X github.com/hashicorp/vault/version.GitCommit=$revision -X github.com/hashicorp/vault/version.BuildDate=$build_date"
+  # if building locally with enos - don't need to set version/prerelease/metadata as the default from version_base.go will be used
+  ldflags="${ldflags} -X github.com/hashicorp/vault/version.GitCommit=$revision -X github.com/hashicorp/vault/version.BuildDate=$build_date"
+
+  if [[ ${BASE_VERSION+x} ]]; then
+    msg="${msg}, base version ${BASE_VERSION}"
+    ldflags="${ldflags} -X github.com/hashicorp/vault/version.Version=$BASE_VERSION"
+  fi
 
-  if [ -n "$prerelease" ]; then
-    msg="${msg}, prerelease ${prerelease}"
-    ldflags="${ldflags} -X github.com/hashicorp/vault/version.VersionPrerelease=$prerelease"
+  if [[ ${PRERELEASE_VERSION+x} ]]; then
+    msg="${msg}, prerelease ${PRERELEASE_VERSION}"
+    ldflags="${ldflags} -X github.com/hashicorp/vault/version.VersionPrerelease=$PRERELEASE_VERSION"
   fi
 
-  if [ -n "$metadata" ]; then
-    msg="${msg}, metadata ${metadata}"
-    ldflags="${ldflags} -X github.com/hashicorp/vault/version.VersionMetadata=$metadata"
+  if [[ ${VERSION_METADATA+x} ]]; then
+    msg="${msg}, metadata ${VERSION_METADATA}"
+    ldflags="${ldflags} -X github.com/hashicorp/vault/version.VersionMetadata=$VERSION_METADATA"
   fi
 
   # Build vault
   echo "$msg"
-  pushd "$(repo_root)"
-  mkdir -p dist
-  mkdir -p out
-  set -x
-  go build -v -tags "$GO_TAGS" -ldflags "$ldflags" -o dist/
-  set +x
-  popd
-}
-
-# Bundle the dist directory into a zip
-function bundle() {
-  : "${BUNDLE_PATH:=$(repo_root)/vault.zip}"
-  echo "--> Bundling dist/* to $BUNDLE_PATH"
-  zip -r -j "$BUNDLE_PATH" dist/
+  go build -o "$BIN_PATH" -tags "$GO_TAGS" -ldflags "$ldflags" -buildvcs=false
 }
 
 # Prepare legal requirements for packaging
 function prepare_legal() {
   : "${PKG_NAME:="vault"}"
-
-  pushd "$(repo_root)"
```

Comment:
Does prepare legal still work without dropping into the root of the repo? Feels like the relative paths in the function would end up putting the legal documents in the scripts directory instead of the `dist` dir that we zip.